ingest/ingest_pipeline.py

```diff
@@ -546,8 +479,8 @@ def exit_pipeline(ingest, status, status_cell_metadata, arguments):
                             'errors.txt',
                             f'parse_logs/{study_file_id}/errors.txt',
                         )
-                    # Need 1 argument that has a path to identify google bucket
-                    # Break after first argument
+                    Need 1 argument that has a path to identify google bucket
+                    Break after first argument
```

Comment:
Blocker: leaving these lines uncommented looks like a functional bug.  Seemingly trivial fix:

```suggestion
                    # Need 1 argument that has a path to identify google bucket
                    # Break after first argument
```